hack/testdata/rollingupdate-statefulset.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: nginx
+spec:
+  updateStrategy:
+    type: RollingUpdate
+  serviceName: "nginx"
+  replicas: 0
+  template:
+    metadata:
+      labels:
+        app: nginx-statefulset
+    spec:
+      terminationGracePeriodSeconds: 5
+      containers:
+      - name: kubernetes-pause
+        image: gcr.io/google-containers/pause:latest
```

Comment:
Use `gcr.io/google_containers/nginx-slim:0.7` for this and `gcr.io/google_containers/nginx-slim:0.8` for rv2 (since the name is nginx)